README.md

```diff
@@ -49,6 +49,33 @@ This list is just for Red Hat contributions to projects hosted on GitHub.  See h
 ### Hosting projects on /RedHatOfficial
 
 Open an issue in *this* repo, and the owners of the RedHatOfficial organization will get in touch with you.
+
+## Quick start
+    ```
+    git clone https://github.com/RedHatOfficial/RedHatOfficial.github.io.git
+    cd RedHatOfficial.github.io
+    
+    ```
+ 
+## Building and Running Locally
+Building and running the Red Hat Official Project Page in your local dev environment is very easy. Be sure you have [Node.js](https://nodejs.org/) installed, then follow the directions below. 
+
+1. Setup Webpack and babel-loader modules using npm
+
+   `npm install babel-loader webpack --save-dev`
```

Comment:
All the required packages are defined in the package.json.  That means you do not need to install dependencies fresh to run the development environment, you only need to fire `npm install` and that will look at the `package.json` and determine what to install.  To that end, you can combine steps 1 & 2 here into just:

```
1. Install dependencies: `npm install`
```